sys/ddb/ddb.h

```diff
@@ -83,7 +83,9 @@ int	DB_CALL(db_expr_t, db_expr_t *, int, db_expr_t[]);
  * Most users should use db_fetch_symtab in order to set them from the
  * boot loader provided values.
  */
-extern vm_offset_t ksymtab, kstrtab, ksymtab_size, ksymtab_relbase;
+extern vm_pointer_t ksymtab, kstrtab;
+extern vm_offset_t ksymtab_relbase;
+extern vm_size_t ksymtab_size;
```

Comment:
```suggestion
extern vm_size_t ksymtab_size;
extern vm_offset_t ksymtab_relbase;
```
keep them in the same order?

Reply:
Done